lib_static/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -438,7 +438,7 @@ def custom_field_values_cache
           @custom_field_values_cache ||= {}
         end
 
-        module ClassMethods
```

Comment:
It's sort of a convention to call the module that adds class methods `ClassMethods` see https://github.com/search?q=%22module+ClassMethods%22+language%3Aruby&type=code. Why are we changing it?

Reply:
I wrote in the PR description, because there two separate `ClassMethods` modules in one file, so one keeps the name and other gets a different one to disambiguate